test/acceptance/support/azure-client.js

```diff
@@ -0,0 +1,56 @@
+const { BlobServiceClient } = require('@azure/storage-blob')
+const { ServiceBusClient } = require('@azure/service-bus')
+const config = require('./config')
+let containerCreated = false
+
+const blobServiceClient = BlobServiceClient.fromConnectionString(`${config.connectionStr}`)
+const container = blobServiceClient.getContainerClient(`${config.container}`)
+const connectionString = `Endpoint=sb://${config.host}/;SharedAccessKeyName=${config.username};SharedAccessKey=${config.password}`
+
+const initialiseContainer = async () => {
+  if (!containerCreated) {
+    await container.createIfNotExists()
+    containerCreated = true
```

Comment:
If statement is redundant as `createIfNotExists` is doing the exact same check before creating the container.